app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt

```diff
@@ -333,6 +335,22 @@ class AdminAuthActivityTest {
     }
   }
 
+  @Test
+  fun testAdminAuthActivity_buttonState_isDisabled() {
```

Comment:
Suggest clarifying in the test name the condition under which it's disabled (to also contrast with the version of this test you could add to verify that it's correctly enabled at the right time).

Reply:
Renamed to `testAdminAuthActivity_defaultButtonState_isDisabled `

And added one more test `testAdminAuthActivity_inputPin_buttonStateIsEnabled `